tenants-certified-attributes-updater/src/test/scala/it/pagopa/interop/tenantscertifiedattributesupdater/CreateActionSpec.scala

```diff
@@ -16,62 +16,70 @@ import java.util.UUID
 
 class CreateActionSpec extends FunSuite {
   test("Create new Tenant with attributes if Tenant does not exist") {
-    val attributeOrigin1 = "IPA"
-    val attributeCode1   = "CAT1"
-    val attributeOrigin2 = "IPA"
-    val attributeCode2   = "CAT2"
+    val origin1        = "ORIGIN_1"
+    val origin2        = "ORIGIN_2"
+    val originId1      = "001"
+    val originId2      = "002"
+    val attributeCode1 = "CAT1"
+    val attributeCode2 = "CAT2"
 
     val institutions: List[Institution]           =
       List(
-        institution("IPA", "001", attributeCode1),
-        institution("IPA", "001", attributeCode2),
-        institution("IPA", "002", attributeCode2)
+        institution(origin1, originId1, attributeCode1),
+        institution(origin2, originId1, attributeCode2),
+        institution(origin1, originId2, attributeCode2)
```

Comment:
⚠️ you have changed test cases.
When refactoring the code, the regression tests help you understand if the outcome is the same (aka the business logic has not been changed).
I suggest to keep the old test cases to verify that unexpected behaviours have not been included.

Additional test cases are always welcome, but the existing should not be changed, if the business logic has not been changed

Reply:
These tests were changed because the `tenantSeed` no longer accepts a single `AttributeInfo` but a list of `AttributeInfo`, so @beetlecrunch had to propagate this change to the tests.

Reply:
that change makes totally sense, my point is that the old set of tests was:
- tenant1, attribute1
- tenant1, attribute2
- tenant2, attribute2

Now the new set is
- tenant1, attribute1
- tenant2, attribute2
- tenant3, attribute2

here you are missing the test case where more than 1 attribute is added to the same tenant